pkg/domain/infra/abi/containers.go

```diff
@@ -67,7 +62,7 @@ func (ic *ContainerEngine) ContainerWait(ctx context.Context, namesOrIds []strin
 	var (
 		responses []entities.WaitReport
 	)
-	ctrs, err := getContainersByContext(false, options.Latest, namesOrIds, ic.Libpod)
+	ctrs, err := shortcuts.GetContainersByContext(false, options.Latest, namesOrIds, ic.Libpod)
```

Comment:
my understanding was we were avoiding the use of shortcuts...

Reply:
Very bad, bad merge.  Correct files now uploaded.